homeassistant/components/efesto/climate.py

```diff
@@ -0,0 +1,284 @@
+"""Support for Efesto heating devices."""
+import logging
+
+import efestoclient
```

Comment:
Only import the used classes instead of importing the entire library.

Reply:
Done